### PR TITLE
[Development] update robot config for navigation/perception

### PR DESCRIPTION
**Pull request by Zhouxunzhe**
## Motivation and Context

Change the dummy agent and manipulation config for randomly choosing actions `pick`, `place`, `nav_to_obj`.

## How Has This Been Tested

This is automatically tested through running `run.py`.

## Types of changes

- **\[Docs change\]** Addition or changes to the documentation
- **\[Refactoring\]** Large changes to the code that improve its functionality or performance
- **\[Dependency Upgrade\]** Upgrades one or several dependencies in habitat
- **\[Bug Fix\]** (non-breaking change which fixes an issue)
- **\[Development\]** A pull request that add new features to the [habitat-lab](/habitat-lab) task and environment codebase. Development Pull Requests must be small (less that 500 lines of code change), have unit testing, very extensive documentation and examples. These are typically new tasks, environments, sensors, etc... The review process for these Pull Request is longer because these changes will be maintained by our core team of developers, so make sure your changes are easy to understand!
- **\[Experiment\]** A pull request that add new features to the [habitat-baselines](/habitat-baselines/) training codebase. Experiments Pull Requests can be any size, must have smoke/integration tests and be isolated from the rest of the code. Your code additions should not rely on other habitat-baselines code. This is to avoid dependencies between different parts of habitat-baselines. You must also include a README that will document how to use your new feature or trainer. **You** will be the maintainer of this code, if the code becomes stale or is not supported often enough, we will eventually remove it.

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation if required.
- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [x] I have added tests to cover my changes if required.

### CODE REVIEW

File: habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone.yaml [hunk 1]
```diff
@@ -141,7 +141,7 @@ habitat:
       agent_0:
         joint_start_noise: 0.0
   dataset:
-    data_path: data/datasets/manipulation/manipulation_eval.json.gz
```
Comment (SgtVincent) [inline habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone.yaml:144]: Have you updated the data locally? I don't have manipulation_eval_test.json.gz .

**Review by SgtVincent — CHANGES_REQUESTED** [review]

File: habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone.yaml (hunk 1 above)
Comment (SgtVincent) [reply habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone.yaml:144]: I you do want to change the data-related config,  please upload your data to google drive for the reviewer to test.

Comment (Zhouxunzhe) [reply habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone.yaml:144]: Solved. I will remember to upload data for test the next time. Thank you for your kind advice!

Comment (SgtVincent) [reply habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone.yaml:144]: Please also change the dataset explanation in the habitat-mas remade accordingly. 

https://github.com/SgtVincent/habitat-lab/blob/cae45df26fdc3ae25e4093f5762007dcb9c89d00/habitat-mas/README.md#L63-L92

Comment (Zhouxunzhe) [reply habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone.yaml:144]: > Please also change the dataset explanation in the habitat-mas remade accordingly.
> 
> https://github.com/SgtVincent/habitat-lab/blob/cae45df26fdc3ae25e4093f5762007dcb9c89d00/habitat-mas/README.md#L63-L92

Done. The new datasets is placed in data/datasets/manipulation/manipulation_eval.json.gz, so the datasets explanation could remain the same. Nothing has to be changed.

**Review by SgtVincent — APPROVED** [review]
Thanks. LGTM!

**Pull request merged**